Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Menu/EditorMenu.cpp

```diff
@@ -116,6 +125,17 @@ namespace AzToolsFramework
         return menuIterator->second;
     }
 
+    AZStd::optional<int> EditorMenu::GetWidgetSortKey(const AZStd::string& widgetActionIdentifier) const
+    {
+        auto widgetIterator = m_widgetToSortKeyMap.find(widgetActionIdentifier);
+        if (widgetIterator == m_widgetToSortKeyMap.end())
+        {
+            return AZStd::nullopt;
+        }
+
+        return widgetIterator->second;
```

Comment:
not sure if it would be better to use a ternary?
```
return (widgetIterator != m_widgetToSortKeyMap.end()) ? widgetIterator->second : AZStd::nullopt;
```